app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,196 @@
+# frozen_string_literal: true
+
+#  News Articles
+#
+#  Actions
+#
+#    create:           Create article from data in "new" form
+#    destroy::         Destroy article
+#    edit::            Display form for editing article
+#    index::           List all articles
+#    index_articles::  List selected (based on last search) articles
+#    new::             Display form for new article
+#    show::            Show one article
+#    update::          Update article from "edit" form
+#
+#
+#  Callbacks and Methods
+#
+#    ignore_request_unless_permitted:: Unless user permitted to perform request,
+#                       then index_articles
+#    permitted?         boolean: permitted to create/update/destroy Articles
+#
+class ArticlesController < ApplicationController
+  before_action :login_required, except: [
```

Comment:
This offense may go away on its own: 
- I may get rid of aliases;
- As @nwilson-eol suggested, it may be unnecessary to includes the aliases in the callbacks.